### DISCUSSION TITLE
Future of the project

### DISCUSSION THREAD

**Discussion by danieldevine** [danieldevine/bird-elephant#51] [Announcements]
[Here are my thoughts on where this project goes from here, following the changes to the Twitter API](https://coderjerk.com/gifting-labour-to-big-business/)

**Comment by developersteve** [comment]
I think anyone thats been in tech even a few years (and I've been in it many) gets the conflict that happens at times, particularly around how open source libraries/projects are used by capitalist businesses who questionably conduct themselves (potentially even instances of governments at times). 

I for one appreciate the work you do to contribute and wish i could contribute more to this and all the many other open source projects that do amazing work

... tip of the hat <3

**Reply by danieldevine** [reply]
Thanks Steve, and thanks for your contributions.
